### PR TITLE
Datasets

### PR DESCRIPTION
**Pull request by proskurin**
# Description

Add financial datasets load functions

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] test_datasets.py

**Test Configuration**:
* Operating system
* IDE used

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: docs/source/getting_started/datasets.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
+
+========
+Datasets
+========
+
+Mlfinlab package contains various financial datasets which can be used by a researcher as sandbox data.
+
+Tick sample
+***********
```
Comment (Jackal08) [inline docs/source/getting_started/datasets.rst:9]:
```
==
H1
==

H2
##

H3
***
```

Comment (Jackal08) [reply docs/source/getting_started/datasets.rst:9]: Please correct your headers.

File: docs/source/getting_started/datasets.rst [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+.. autofunction:: load_tick_sample
+
+
+Dollar bars sample
+******************
+
+We also provide a sample of dollar bars for E-Mini S&P 500 futures. Data set structure:
+
+    - Open price (open)
+
```
Comment (Jackal08) [inline docs/source/getting_started/datasets.rst:24]: Dont need white spaces between bullets.

File: docs/source/getting_started/datasets.rst [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+
+ETF prices
+**********
+
+.. py:currentmodule:: mlfinlab.datasets.load_datasets
+.. autofunction:: load_dollar_bar_sample
+
+The data set consists of close prices for EEM,EWG,TIP,EWJ,EFA,IEF,EWQ,EWU,XLB,XLE,XLF,LQD,XLK,XLU,EPP,FXI,VGK,VPL,SPY,TLT,BND,CSJ,DIA starting from 2008 till 2016.
```
Comment (Jackal08) [inline docs/source/getting_started/datasets.rst:53]: Add colon: `prices for:`

File: docs/source/getting_started/datasets.rst [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+
+Data sets load example
+**********************
+
+.. code-block::
+
+	from mlfinlab.datasets import (load_tick_sample, load_stock_prices, load_dollar_bar_sample)
+
+	  tick_df = load_tick_sample()
```
Comment (Jackal08) [inline docs/source/getting_started/datasets.rst:64]: Strange indent. 

Please note that you must not use tabs but rather 3 whitespaces. ` `

.rst treats them differently

File: mlfinlab/datasets/__init__.py [hunk 5]
```diff
@@ -0,0 +1,4 @@
+"""
```
Comment (Jackal08) [inline mlfinlab/datasets/__init__.py:1]: Did you remember to also expose this in the mlfinlab/__init__.py file?

File: mlfinlab/datasets/data/tick_data.csv [hunk 6]
```diff
@@ -0,0 +1,101 @@
+Date and Time,Price,Volume
```
Comment (Jackal08) [inline mlfinlab/datasets/data/tick_data.csv:1]: This file is very small?

Comment (proskurin) [reply mlfinlab/datasets/data/tick_data.csv:1]: Yep, I took it from test_data

**Pull request merged**